theme/templates/latex.tpl

```diff
@@ -173,10 +174,10 @@
 
 ((* block maketitle *))
 
+\title{((( nb.metadata.get("latex_metadata", {}).get("title", "") | escape_latex )))}
+
 ((*- if nb.metadata.get("latex_metadata", {}).get("author", ""): -*))
 \author{((( nb.metadata["latex_metadata"]["author"] )))}
-((*- else -*))
-\author{Al Pacino}
```

Comment:
@AakashGfude Can you elaborate on this change? I think we should credit Al Pacino for his contributions to the theme, if appropriate `:-P`